### PR TITLE
Træfik implementation and small usability fixes

### LINKED ISSUE #40: Service should be masked behind reverse proxy
Due to some security limitations of Keycloak raised by @inbarbarkai  we may need to implement a reverse proxy for our service in addition to the other dockers.

There's an option to already implement Træfik solution for development environment instead of nginx and benefit from the extended monitoring capabilities and ease of use:
https://dev.to/flemssound/development-enviroment-with-docker-and-traefik-1lg6

### PR DESCRIPTION
**Pull request by yarons**
A fairly complex fix that masks all the services behind a single endpoint that can be diverted upon request.

Currently we have:

- invitease.localhost
- keycloak.localhost
- dash.localhost

All of them can be converted to:

- invitease.localhost
- invitease.localhost/keycloak
- invitease.localhost/dash

Binding these all under the same domain as required by @inbarbarkai 

Closes #40

### CODE REVIEW

File: src/routers/inviter_router.py [hunk 1]
```diff
@@ -1,3 +1,12 @@
...
 
 router = APIRouter(prefix="/inviter", tags=["inviter"])
+
+def create_invitations_list():
+    connection = create_connection()
+    return InvitationRepository(connection)
+
+@router.get("/invitations", summary="Gets all the invitations for this user.")
+def get_invitations(invitations: InvitationRepository = Depends(create_invitations_list), user_id: str = None):
+    return invitations.query(lambda x: x.filter(Invitation.user_id == user_id))
```
Comment (inbarbarkai) [inline src/routers/inviter_router.py:12]: We agreed this login should be in the repository

Comment (yarons) [reply src/routers/inviter_router.py:12]: Indeed, consider this a test, we're not going to build anything on it.

**Comment by yarons** [comment]
https://github.com/flexguse/traefik-inter-container-routing

We can use that for the sake of a publicly known unique name.

**Comment by inbarbarkai** [comment]
Looks promising...
We need the container to know keycloak like the host does.

**Pull request merged**